flask_resty/pagination.py

```diff
@@ -542,10 +560,28 @@ def get_filter_clause(self, column_cursors):
             value = int(value)
 
         if asc:
-            current_clause = column > value
+            if value is None:
+                return None
+            elif value is not None:
+                current_clause = self._handle_nullable(
+                    column, value, is_nullable
+                )
+            else:
+                current_clause = column > value
         else:
-            current_clause = column < value
+            if value is None:
+                current_clause = column.isnot(None)
+            else:
+                current_clause = column < value
+        return current_clause
+
+    def get_filter_clause(self, column_cursors):
+        previous_clauses = self.get_previous_clause(column_cursors[:-1])
+        column, asc, value = column_cursors[-1]
 
+        current_clause = self._prepare_current_clause(column, asc, value)
+        if not str(previous_clauses):
```

Comment:
what is this checking?

Reply:
After adding `isnot_distinct_from` this line is unnecessary